iceoryx_posh/include/iceoryx_posh/internal/roudi/roudi_multi_process.hpp

```diff
@@ -41,33 +41,33 @@ using namespace iox::units::duration_literals;
 class RouDiMultiProcess
 {
   public:
-
     // indicate whether the message queue thread will start directly or deferred
     // this is important for derived classes which may need to initialize their members before the thread starts
-    enum class MQThreadStart {
-      IMMEDIATE,
-      DEFER_START
+    enum class MQThreadStart
+    {
+        IMMEDIATE,
+        DEFER_START
     };
-   
+
 
     RouDiMultiProcess& operator=(const RouDiMultiProcess& other) = delete;
     RouDiMultiProcess(const RouDiMultiProcess& other) = delete;
 
-    RouDiMultiProcess(RouDiMemoryInterface& roudiMemoryInteface,
-                      PortManager& portManager,
-                      const MonitoringMode f_monitoringMode = MonitoringMode::ON,
-                      const bool f_killProcessesInDestructor = true,
-                      const MQThreadStart mqThreadStart = MQThreadStart::IMMEDIATE);
+    RouDiMultiProcess(
+        RouDiMemoryInterface& roudiMemoryInteface,
+        PortManager& portManager,
+        const MonitoringMode f_monitoringMode = MonitoringMode::ON,
```

Comment:
This could all be moved to the RouDiConfig. Would be ideal for someone who wants to start contributing to iceoryx.We  need to create an issue for this.

Reply:
@sculpordwarf Are you interested?

Reply:
@dkroenke Not in this pull. This depends on code not changed logically by me -> clean-up of a side topic -> new ticket for that issue.